go.mod

```diff
@@ -3,8 +3,7 @@ module github.com/wavefronthq/go-metrics-wavefront
 go 1.13
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.5.1
```

Comment:
Will you please explain why you can't use the latest version of testify?

Reply:
Their travis buils against very old versions of golang that do not support 1.7. the question is why are they still testing against golang 1.13?

Reply:
ok thanks for fixing this.